### DISCUSSION TITLE
Sensor Rollout process

### DISCUSSION THREAD

**Discussion by rosswakelin** [Security-Onion-Solutions/securityonion#6679] [Unsupported Versions] [answered]
We are considering rolling out about 25 IDS sensor devices to sites across the country.  We are looking at "building" the sensors at our central site, customising for each site, and then shipping pre-configured boxes.  
Has anyone come up with a process for working out everything that has be changed on a forwarder when it is moved to a new location/subnet etc.

**Comment by dougburks** [comment, marked as answer]
This isn't supported. We recommend configuring boxes with their final IP address as that is used to automatically create firewall rules on the manager to allow communication.
